packages/wrangler/src/type-generation/index.ts

```diff
@@ -511,6 +512,12 @@ export async function generateEnvTypes(
 		}
 	}
 
+	if (configToDTS.pipelines) {
+		for (const pipeline of configToDTS.pipelines) {
+			envTypeStructure.push([constructTypeKey(pipeline.binding), "Pipeline"]);
```

Comment:
This won't work, sine the `Pipeline` type isn't available globally anymore, as far as I know. This will need to be `import("cloudflare:pipelines").Pipeline`

Reply:
Updated!